src/renderer/Dataframe.js

```diff
@@ -497,6 +497,20 @@ export default class Dataframe extends DummyDataframe {
         const getters = {};
         for (let i = 0; i < this.metadata.propertyKeys.length; i++) {
             const propertyName = this.metadata.propertyKeys[i];
+            if (this.metadata.properties[propertyName].aggregations) {
```

Comment:
I think this could be refactored to a new function called **_buildGetters**, with an objective of populating the `getters` object.

It could be something like:
```suggestion
 const metadata = this.metadata;
 const dataframe = this._dataframe;
 const getters = this._buildGetters(metadata, dataframe);
 
_buildGetters (metadata, dataframe) {
    // code with the for...
}
```

It could also have no input params, as both are available through _this_.

Reply:
:+1: